cmd/tracee-ebpf/internal/config/config.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type TraceeEbpfCliConfig struct {
```

Comment:
I think `TraceeEbpf` is redundant here, since we are in `cmd/tracee-ebpf`

Reply:
That's fair, do you think we should even export the struct itself? So change it to `cliConfig`?

Reply:
Renamed to `CliConfig`